deployment/helm/skaha/sample-local-values.yaml

```diff
@@ -44,19 +49,19 @@ deployment:
 
     # Optionally mount a custom CA certificate
     extraVolumeMounts:
-    # - mountPath: "/config/cacerts"
-    #   name: cacert-volume
+     - mountPath: "/config/cacerts"
+       name: cacert-volume
```

Comment:
Please omit this.  It's optional for users.

Reply:
Done.